src/pages/GetInvolved/Club.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { withRouter, Link } from "react-router-dom";
+import Layout from "components/Layout/Layout";
+
+const Club = ({ match }) => {
+  let clubDataAll = useSelector((state) => state.clubs);
+  let currentClub = `${match.params.clubId}`;
+  let clubData = useSelector((state) =>
+    clubDataAll.find((object) => object.slug === currentClub)
+  );
+
```

Comment:
You may want to consider destructuring the object so you dont have todo this nested dot notation

example: 
```
const { club_description, club_image, etc. } = clubData.acf.club;
```